frontend/src/pages/SignupPage/index.js

```diff
@@ -4,11 +4,12 @@ import SocialMediaButton from "components/SocialMediaButton";
 import Title from "components/Typography/Title";
 
 import styles from "./index.module.css";
+import * as constants from "constants/Subvisual.js";
```

Comment:
You don't have to import everything from the constants file, if all you need is, for example, the SUBVISUAL constant.

Also, consider object destructuring here too.

Reply:
We will :+1: